client/my-sites/google-my-business/index.js

```diff
@@ -36,6 +29,15 @@ export default function() {
 			makeLayout,
 			clientRender
 		);
+
+		page(
+			'/google-my-business/stats/:site',
```

Comment:
This makes sense 👍

Reply:
I thought we could define the route in the `Stats` router, but this is probably not a good idea since that would make the `Google My Business` section a dependency of `Stats` (it wouldn't be efficient to load all the code from the `Google My Business` section whenever a user enters the `Stats` section given that only a very small amount of users will enable this feature).